ably/README.md

```diff
@@ -0,0 +1,64 @@
+## Overview
+The [Ably][1] platform is used to power real-time use cases such as multiplayer, chat, data synchronization, data broadcast, and notifications for highly scalable web and mobile applications around the world. Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
+
+Using Ably's Datadog Integration, you can:
+- Use [Ably statistics][2] alongside other key metrics in Datadog
+- Correlate Ably message, channel, and connection usage for collaborative analysis in Dashboards
+- Identify unexpected activity and troubleshoot potential issues with your applications powered by Ably
+
+
+## Setup
+
+- **In Datadog**: Go to **Integrations**, select the Ably tile and click **Install Integration**.
+
+- **In Ably**: Go to https://ably.com, log in and navigate to **Your Apps**.
+
+![Ably Screenshot][3]
+
+- Select the **Ably App** you would like to set up the **Datadog Integration** for and click **Integrations**.
+
+![Ably Screenshot][4]
+
+- Click the **Connect to Datadog** button.
+
+![Ably Screenshot][5]
+
+Your Ably App statistics now appear in Datadog.
+
```

Comment:
Once you have access to the developer platform, please update the instructions for oauth. Thanks!